lxd/images.go

```diff
@@ -881,9 +881,9 @@ func doImagesGet(d *Daemon, recursion bool, project string, public bool) (interf
 
 func imagesGet(d *Daemon, r *http.Request) Response {
 	project := projectParam(r)
-	public := d.checkTrustedClient(r) != nil
+	trusted, _ := d.checkTrustedClient(r)
```

Comment:
This seems a regression in code clarity: the former code gave a slightly more clear hint that in case the user is not trusted only public images should be returned.